sandy-city-utah/active-directory-api
# feat(package): use shared sandy libs and fix concurrency bug

### What Changed?
  - add npmrc designating the registry to be GitHub
  - install shared packages and update the references in the codebase
  - update readme and build file to reflect the changes
  - fix nasty bug that caused the process to miss some AD entries; make async queue

### Issues Addressed:
  - P21312-19

### Issue Testing:
  - Manual Developer Testing

### Test Results:
  1. Ensure you have logged in to the GitHub package registry
  1. Attempt to install the packages
  1. See that it installed the packages successfully
  1. Run the application
  1. See that it starts up correctly as before
  1. Hit the users endpoint from another application
  1. Watch the process complete
  1. See that the count of users returned from the API is ~476

### Additional Information:
In order to install pacakges, you must authenticate with the GitHub package manager by using `npm login` as mentioned in the readme
